test/Feature/AddressOfLabels.c

```diff
@@ -0,0 +1,35 @@
+// RUN: %llvmgcc -emit-llvm -g -c %s -o %t.bc
+// RUN: rm -rf %t.klee-out
+// RUN: %klee --output-dir=%t.klee-out %t.bc 2> %t.log
+// RUN: grep "cannot handle constant 'i8\* blockaddress" %t.log
+// RUN: grep  "AddressOfLabels.c:21" %t.log
```

Comment:
Please use `FileCheck` here. Doing so is less fragile because we can avoid hard coding the line number by doing

```
// CHECK: cannot handle constant i8* blockaddress at location AddressOfLabels.c:[[@LINE:+1]]
```